src/Web/Hyperbole/HyperView.hs

```diff
@@ -32,6 +32,10 @@ instance HyperView Message where
 -}
 class (Param id, Param (Action id)) => HyperView id where
   type Action id :: Type
+  hyperViewMod :: id -> Mod
```

Comment:
Can you tell me how this came up? Could it have been solved by nesting a `col` inside of a `hyper`?